src/color_wheel.cpp

```diff
@@ -321,6 +365,19 @@ void ColorWheel::paintEvent(QPaintEvent * )
     QPointF h2 = ray.p2();
     painter.drawLine(h1,h2);
 
+    for (auto const& editor : p->ring_editors)
+    {
+        // TODO: separate function
+        painter.setPen(QPen(Qt::white,3));
```

Comment:
It might be a good idea to have different colors for the ones which are not editable. If you mix editable and not editable harmonies, it becomes a bit confusing to use.